### PR TITLE
Added automated available skins configuration data using files from directory instead of manually editing index.html

### PR DESCRIPTION
**Pull request by TheGiga**
what title says

### CODE REVIEW

**Review by drippycatcs — APPROVED** [review]
Looks good 👍

**Comment by drippycatcs** [comment]
Dynamically load cone skins from files

File: app.js [hunk 1]
```diff
@@ -89,6 +89,19 @@ app.get('/api/leaderboard/:name', async (req, res, next) => {
...
 });
 
+app.get('/api/skins/available', async (req, res) => {
+    let availableSkins = {}
+
+    let files = await fs.readdir(`${process.cwd()}/public/skins`)
+
+    files.forEach(file => {
+        if (file.endsWith('.png') && file.startsWith('cone_'))
+            availableSkins[file.substring(5, file.length -4)] = `/skins/${file}`;
```
Comment (SyberiaK) [inline app.js:99]: Wait, doesn't it make more sense to define all skins in some json or yaml file when we define [their odds](https://github.com/drippycatcs/coneflip-overlay/blob/abb22a8a1dde26453018fc332a28263376509ad2/app.js#L16) manually anyway?

Comment (TheGiga) [reply app.js:99]: never noticed this existed lol, well, some fallback option should be added as a default, when the skin is not defined here, but, at least, that's one less place where you should manually write in stuff.

**Pull request merged**